docs/proxy-keys/proxy-key-sols.tex

```diff
@@ -0,0 +1,319 @@
+  \item This solution requires the full proxied balance to be delegated by the \gls{trustee}. 
+  \item With a Ledger based approach consider that once \emph{any} proxy delegation certificates are posted, and the \gls{trustor} wishes to spend some amount $>n$, where $n$ is the balance which is \emph{not} proxied, we must take some undesirable action, such as:
+  \begin{itemize}
+    \item Immediately invalidate all proxied rights.
+    \item Reissue all proxy delegation certificates with a new notional, reduced proportionally across all \glsplural{trustee}.
+    \item Lock funds until manual action is taken. 
+  \end{itemize} 
+  \item Granular control over generated rewards is difficult to manage in a Ledger solution context, without further metadata being added to the proxy delegation certificates and a radical change to how the Ledger handles reward flows.
+  \item A Ledger based solution will still require interactivity, i.e. sharing of the $\mathcal{A}_{p,\text{target}}$ public key/hash. To minimise interactivity we could define a request/response mechanism, however this would open a possible DDOS attack vector. 
+  \item Privacy is difficult to achieve in a Ledger solution context. The chain is transparent by design and a Ledger based solution requires posting data to the chain. Privacy would require implementation of a zero knowledge proving/verification system between all parties using the chain.
+  \item A Ledger based solution does not avoid changes to the Wallet. 
+\end{enumerate}
+
+\pagebreak
+
+\subsection{A Wallet based approach}
+Conceptually, the Wallet based solution to deliver Proxy-keys is predicated on the notion of creating a 
+\gls{joint address} comprised of both a \gls{trustor} and \gls{trustee} component. Let us remind ourselves how 
+a Cardano \emph{Shelley}-era address is generated:
+
+\begin{figure}[ht]
+  \includegraphics[width=\linewidth]{images/addresses.pdf}
+  \caption{address/credential interaction - from SL-D1}
+  \label{fig:addresses}
+\end{figure}
+
+As shown above the structure of a Shelley-era address (from which one can delegate stake), is forged from a 
+\emph{payment credential}, $\mathcal{A}_p$ and a \emph{stake credential}, $\mathcal{A}_s$. With the current 
+core Cardano protocol, the payment credential, (or more accurately the credential's private key counterpart), 
+controls the spending of funds, and the stake credential controls delegation/de-delegation and withdrawal of 
+rewards from the account they accumulates to. \\
+
+\subsubsection{High-level design}
+Cognisant of the above, the solution to deliver the Proxy-key/partial delegation feature is to deposit funds
```

Comment:
I'm guessing this is the British spelling :)

Reply:
Do you mean best spelling 😆

Reply:
:laughing: